nussknacker-dist/src/universal/conf/dev-application.conf

```diff
@@ -307,5 +307,7 @@ countsSettings {
   database: "esp"
 }
 
+codeSuggestionsFromBackend: false
```

Comment:
Let's remove it and use env variables instead. Take a look for the simplest way to run nussknacker - oneliner available here: https://github.com/TouK/nussknacker#quickstart. It should be possible to turn it on by adding `-e ...=true`

Reply:
Done